src/pages/Calendar/Calendar.tsx

```diff
@@ -35,14 +78,63 @@ const Calendar = () => {
             Historic
           </p>
         </div>
-        <img
-          className="calendar__image"
-          src={calendarImg}
-          alt="Calendar image"
-        />
+
+        <div className="calendar__container">
+          {isActive && (
+            <Calendar
+              value={selectedDay}
+              onChange={setSelectedDay}
+              colorPrimary="#b42004"
+              calendarSelectedDayClassName="calendar__day"
+            />
+          )}
+        </div>
+        {isActive && (
+          <div className="displayed-events">
+            {filteredSearch.map((event: Event, index: number) => {
+              return (
+                <EventCard
+                  key={event.id}
+                  title={event.name}
+                  maker={event.category}
+                  date={event.date}
+                  textContent={event.description}
+                  galleryArray={event.images}
+                  buttonLabel={
+                    buttonVariants[index] ? "CANCEL BOOKING" : "BOOK A PLACE"
+                  }
+                  buttonVariant={buttonVariants[index]}
+                  handleClick={() => handleClickButton(index)}
+                />
```

Comment:
I think that you should handle it when you are isActive and there aren't any events on that date. 

This can just be a message saying something along the lines of "There are currently no events happening on this date."

Reply:
Just checking - do you mean that a message should appear when a user clicks a calendar date that doesn't have an event on?